src/main/java/seedu/contax/logic/parser/CliSyntax.java

```diff
@@ -20,6 +20,8 @@
     public static final Prefix PREFIX_RANGE_FROM = new Prefix("from/");
     public static final Prefix PREFIX_RANGE_TO = new Prefix("to/");
     public static final Prefix PREFIX_EQUALS = new Prefix("=/");
+    public static final Prefix PREFIX_START_WITH = new Prefix("^/");
+    public static final Prefix PREFIX_END_WITH = new Prefix("$/");
```

Comment:
Could you use a word or alphabet or abbreviation instead? The symbols `$` and `^` are not universally understood, unlike `=`.

Reply:
thanks, will change to abbreviation